src/routes/chat/Chat.container.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+
+import { fetchChatListAction } from './Chat.ducks';
+import Chat from './Chat';
+
+const matchState = state => {
```

Comment:
Not going to use them anymore if we stop using connect, but as said before in the previous PR,........ `mapStateToProps` is the convention.